### ISSUE TITLE
Could I specify the port for the proxy?

### ISSUE THREAD

**Issue by gitercn** [Corsinvest/cv4pve-pepper#10]
I run my Proxmox VE using frp reverse proxy for remote access. The local web gui page https://192.168.123.182:8006/ is redirected to https://43.249.xxx.xxx:55012 and the local's 3128 port is also redirected to remote 55011 port.

If I want to use SPICE remotely, I have to download the .vv file and quickly edit the proxy's line from proxy=http://43.249.xxx.xxx:3128 to proxy=http://43.249.xxx.xxx:55011 in able to connect. Since the *.vv file expires in 30 seconds, it's not easy to do so.

So I searched around and found this project, but it seems it can's specify the port for the proxy either. If I run this command, it can fetch the .vv file, but it's still using the 3128 proxy port. 
```
.\cv4pve-pepper.exe --host 43.249.xxx.xxx:55012 --username root@pam --password ****** --vmid 110 --viewer "C:\Program Files\VirtViewer v9.0-256\bin\remote-viewer.exe" --proxy 43.249.xxx.xxx --debug
```

If I run this command by adding the port, it can't fetch the .vv file
```
.\cv4pve-pepper.exe --host 43.249.xxx.xxx:55012 --username root@pam --password ****** --vmid 110 --viewer "C:\Program Files\VirtViewer v9.0-256\bin\remote-viewer.exe" --proxy 43.249.xxx.xxx:55011 --debug
```
It will result in:
```
=============================
Method: POST, Url: https://43.249.xxx.xxx:55012/api2/spiceconfig/nodes/pve5/qemu/110/spiceproxy
Parameters:
proxy : http://43.249.xxx.xxx:55011
StatusCode:          BadRequest
ReasonPhrase:        Parameter verification failed.
IsSuccessStatusCode: False

=============================
```

Could I specify the port for the proxy?

**Comment by gitercn** [comment]
I have created a pull request https://github.com/Corsinvest/cv4pve-pepper/pull/11

**Comment by franklupo** [comment]
Fix in release 1.3.9
